app/controllers/responsible_body/mobile/manual_requests_controller.rb

```diff
@@ -60,4 +60,16 @@ def extra_mobile_data_request_params
       confirm
     ])
   end
+
+  def build_success_message(mno_is_participating)
+    if mno_is_participating
+      I18n.t('responsible_body.extra_mobile_data_requests.create.success')
+    else
+      I18n.t('responsible_body.extra_mobile_data_requests.create.success_saved')
+    end
```

Comment:
(non-blocking suggestion) do you think it might make more sense to rename these keys to something more aligned with the criteria under which they'll be used, rather than the content of the message, like `....success.participating_mno` and `...success.non_participating_mno` ?

Reply:
Nice idea